README.md

```diff
@@ -1 +1,105 @@
-# node-modules-json
\ No newline at end of file
+# Importing JSON files
+
+## Overview
+
+This lesson will cover how and why you would import JSON files.
+
+## Objectives
+
+1. Describe how to import JSON files using provided example: configurations, data, settings
+1. Describe why you'd need to have JSON files
+
+## Reading JSON Files
+
+JSON stands for JavaScript Object Notation. It's a widely use data format. When it comes to Node applications, developers like to use JSON file to store various data such as:
+
+* Configurations
+* Seed data (initial data to populate the development or test database)
+* Settings
+* Plugins for frameworks
+
+
+Let's implement code which reads a JSON file and parses the file data into JavaScript object (`read.js`):
```

Comment:
tell us what the contents of the file is, what data are we parsing? addresses? names? urls?